db/seeds/feeds.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+unless Feed.find_by_slug('fashion-tumblr').present?
+  Feed.create!(
+    name: 'Fashion Tumblr',
+    url: 'http://europeanafashion.tumblr.com/rss'
+  )
+
+  # if the landing page is already present we need to add our feed to it here
```

Comment:
This seems to duplicate the seeding in db/seeds/landing_pages.rb. I think it belongs in a migration rather than the seeds.